plugins/synapticsmst/fu-plugin-synapticsmst.c

```diff
@@ -111,6 +111,7 @@ fu_plugin_synaptics_add_device (FuPlugin *plugin,
 	dev = fu_device_new ();
 	fu_device_set_id (dev, dev_id_str);
 	fu_device_add_flag (dev, FWUPD_DEVICE_FLAG_ALLOW_ONLINE);
+	fu_device_add_flag (dev, FWUPD_DEVICE_FLAG_ALLOW_OFFLINE);
```

Comment:
This one is definitely hardware state dependent.  The dock needs to be plugged in, so if it's scheduled for later I think this could be problematic.